tooling/camel-k-maven-plugin/src/it/generate-catalog/pom.xml

```diff
@@ -30,7 +30,7 @@
     <properties>
         <catalog.path>${project.basedir}</catalog.path>
         <camel.version>3.0.0-RC1</camel.version>
-        <runtime.version>1.0.2-SNAPSHOT</runtime.version>
+        <runtime.version>1.0.5-SNAPSHOT</runtime.version>
```

Comment:
maybe we can use a non snapshot version

Reply:
I'm not very sure as it looks like used in the [camel-k builder_steps](https://github.com/apache/camel-k/blob/59bebae1d9f9d2414979a2c18f9e8ab581c42977/pkg/builder/builder_steps.go#L251)